modules/drivers/mongo/src/metabase/driver/mongo.clj

```diff
@@ -139,25 +139,22 @@
                                  (find-nested-fields field-value nested-fields)
                                  nested-fields)))))
 
-;; TODO - use [[metabase.driver.common/class->base-type]] to implement this functionality
```

Comment:
`metabase.driver.common/class->base-type` is used at line 154, so I'm not sure what this comment is about.